### PR TITLE
Re-merge "Update aws-sdk gem version"

### PR DESCRIPTION
**Pull request by joshlory**
Re-merge PR https://github.com/code-dot-org/code-dot-org/pull/14798.

### CODE REVIEW

File: lib/cdo/aws/cloudfront.rb [hunk 1]
```diff
@@ -258,6 +260,10 @@ def self.config(app, reference = nil)
               s3_origin_config: {
                 origin_access_identity: ''
               },
+              custom_origin_config: {
+                origin_read_timeout: 5,
+                origin_keepalive_timeout: 30
+              },
```
Comment (wjordan) [inline lib/cdo/aws/cloudfront.rb]: these lines need to be removed - only one of `s3_origin_config` _or_ `custom_origin_config` are required.

Comment (joshlory) [reply lib/cdo/aws/cloudfront.rb]: Do I add these options to `s3_origin_config` or remove them entirely?

Comment (wjordan) [reply lib/cdo/aws/cloudfront.rb]: remove entirely.

**Comment by joshlory** [comment]
Updated!  Is there any way to test these changes before the merge to staging?

**Comment by ashercodeorg** [comment]
@joshlory, our pipeline is pretty clean right now (about to get my third green DTT), if you want to merge this, I'm happy to deal with any fallout as DOTD.

**Comment by joshlory** [comment]
The `Update CloudFront` build step initially failed on merge.  Succeeded after adding one more config option in https://github.com/code-dot-org/code-dot-org/commit/5abcb02fb08d8382b0e7533df91868a3567b0a90.

https://codedotorg.slack.com/archives/C03CK8E51/p1494301821954043

File: lib/cdo/aws/cloudfront.rb [hunk 2]
```diff
@@ -245,7 +245,9 @@ def self.config(app, reference = nil)
                 origin_ssl_protocols: {
                   quantity: 2,
                   items: %w(SSLv3 TLSv1)
-                }
+                },
+                origin_read_timeout: 5,
+                origin_keepalive_timeout: 30
```
Comment (wjordan) [inline lib/cdo/aws/cloudfront.rb:250]: @joshlory turns out these values are mismatched, they should have been `origin_read_timeout: 30, origin_keepalive_timeout: 5` to match the defaults. sorry for not picking this up earlier

**Comment by joshlory** [comment]
Also needed one more config option: https://github.com/code-dot-org/code-dot-org/commit/eeaa8afe18d729774b6b09a97f741126fe2bba9a.

**Pull request merged**